apps/learning_nuggets/migrations/0001_initial.py

```diff
@@ -0,0 +1,224 @@
+                                    ],
+                                    "help_text": "You can add the video's transcript here (unlimited characters).",
+                                    "required": False,
+                                },
+                            ),
+                            8: (
+                                "wagtail.blocks.StructBlock",
+                                [
+                                    [
+                                        ("title", 3),
+                                        ("description", 4),
+                                        ("media", 5),
+                                        ("media_type", 6),
+                                        ("transcript", 7),
+                                    ]
+                                ],
+                                {
+                                    "help_text": "Upload or select an instructional video to complement the written content (MP4 format recommended)",
+                                    "required": False,
+                                },
+                            ),
+                            9: (
+                                "wagtail.documents.blocks.DocumentChooserBlock",
+                                (),
+                                {
+                                    "help_text": "Upload related documents such as worksheets, guides, or additional resources (PDF format recommended)",
+                                    "required": False,
+                                },
+                            ),
+                            10: (
+                                "wagtail.blocks.StructBlock",
+                                [
+                                    [
+                                        ("title", 0),
+                                        ("thumbnail", 1),
+                                        ("description", 2),
+                                        ("video", 8),
+                                        ("document", 9),
+                                    ]
+                                ],
+                                {},
+                            ),
+                        },
+                        validators=[
+                            apps.learning_nuggets.models.validate_single_instance
+                        ],
+                    ),
+                ),
+                (
+                    "category",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.SET_NULL,
+                        related_name="nuggets",
+                        to="a4_candy_learning_nuggets.learningcategory",
+                    ),
+                ),
+            ],
+            options={
+                "verbose_name": "Learning Nugget Page",
+            },
+            bases=("wagtailcore.page",),
+        ),
+    ]
```

Comment:
@hom3mad3 I squashed them in one file, by deleting them all, and recreating them.